File: app/src/main/java/com/silver/cat/nilo/config/dagger/ForApplication.java
package com.silver.cat.nilo.config.dagger;

import java.lang.annotation.Retention;

import javax.inject.Qualifier;

import static java.lang.annotation.RetentionPolicy.RUNTIME;

/**
 * Created by xiezhenyu on 2017/1/17.
 */


@Qualifier
@Retention(RUNTIME)
public @interface ForApplication {
}

File: app/src/main/java/com/silver/cat/nilo/config/dagger/ForApplication.kt
package com.silver.cat.nilo.config.dagger

import java.lang.annotation.Retention

import javax.inject.Qualifier

import java.lang.annotation.RetentionPolicy.RUNTIME

/**
 * Created by xiezhenyu on 2017/1/17.
 */


@Qualifier
@Retention(RUNTIME)
annotation class ForApplication
